src/main/groovy/edu/oregonstate/mist/locations/core/Attributes.groovy

```diff
@@ -0,0 +1,26 @@
+package edu.oregonstate.mist.locations.core
+
+class Attributes {
+    String name
+    String abbreviation
+    String latitude
+    String longitude
+    String summary
+    String description
+    String address
+    String city
+    String state
+    String zip
+    String county
+    String telephone
+    String fax
+    List<String> thumbnails
+    List<String> images
+    List<String> departments
+    String website
+    Integer sqft
+    String calendar
+    String campus
+    String type // used for searching. values: building, dining.
+    HashMap<Integer, List<DayOpenHours>> openHours = new HashMap<Integer, List<DayOpenHours>>()
```

Comment:
`openHours` could use the interface `Map` if the implementation as `HashMap` is not relevant to the application programmer.